src/components/Card.js

```diff
@@ -0,0 +1,33 @@
+import { getTimeDifferenceMessage, getDateText } from "../utils/getTimeText";
+import "./Card.css";
+
+export default function Card({ link }) {
+  const parsingDate = new Date(link.createdAt);
+  const timeDifference = Date.now() - parsingDate;
+  const date = getDateText(parsingDate);
+  const timeDifferenceMessage = getTimeDifferenceMessage(timeDifference);
```

Comment:
P2:
해당 부분 모두 `timeDifferenceMessage`를 위한 로직이라 getTimeDifferenceMessage가 link.createAt을 받아 다 처리하도록 하면 좋겠네요.

```suggestion
  const timeDifferenceMessage =  getTimeDifferenceMessage(link.createdAt);
```